nyc_taxi/dashboard.ipynb

```diff
@@ -11,12 +11,12 @@
+    "<img align='right' style='padding:0px 0px 0px 0px;' \n",
+    "src='./thumbnails/index.png'/>\n",
     "\n",
     "The [NYC Taxi trips](https://anaconda.org/jbednar/nyc_taxi/notebook) dataset is a well-studied data science example. Here we show how to build a simple dashboard for exploring 10 million taxi trips in a Jupyter notebook using [Datashader](http://datashader.org), then deploying it as a standalone dashboard using [Panel](http://panel.pyviz.org).\n",
```

Comment:
Oops I was sure I commented that already, but no :) Could you:

- Remove the first link to Jim's old notebook, that doesn't look useful
- Update http to https links
- Update panel.pyviz.org to panel.holoviz.org

?

Reply:
You made the comment initially and I've mostly resolved it.